chrome/content/zotero/xpcom/schema.js

```diff
@@ -1469,7 +1469,7 @@ Zotero.Schema = new function(){
 			});
 			yield _updateDBVersion('compatibility', _maxCompatibility);
 			
-			var sql = "INSERT INTO libraries (libraryID, libraryType, editable, filesEditable) "
+			var sql = "INSERT INTO libraries (libraryID, type, editable, filesEditable) "
 				+ "VALUES "
 				+ "(?, 'user', 1, 1), "
 				+ "(4, 'publications', 1, 1)"
```

Comment:
Why are we specifying library IDs here at all? why not just let DB assign those? (granted that they would pretty much always end up 1 and 2 anyway)